lib/Router/Endpoints/Subscription.php

```diff
@@ -21,6 +25,17 @@ class Subscription {
     'global' => AccessControl::NO_ACCESS_RESTRICTION,
   ];
 
+  function captcha($data) {
+    $subscription = $this->initSubscriptionPage(UserSubscription\Pages::ACTION_CAPTCHA, $data);
```

Comment:
I find this unused variable a bit confusing. Could you please change it just to `$this->initSubscriptionPage(UserSubscription\Pages::ACTION_CAPTCHA, $data);`